tools/crm_error.c

```diff
@@ -120,24 +120,31 @@ main(int argc, char **argv)
             end = 256;
         }
 
-        for (rc = start; rc < end; rc++) {
-            if (rc == (pcmk_rc_error + 1)) {
-                // Values in between are reserved for callers, no use iterating
-                rc = pcmk_rc_ok;
+        current = start;
+        while (current < end) {
+            if (current == (pcmk_rc_error + 1)) {
+                /* Values between here and pcmk_rc_ok are reserved for callers,
+                 * so skip them
+                 */
+                current = pcmk_rc_ok;
+                continue;
```

Comment:
A for loop with some extra bumps inside is actually not that strange, e.g. you see it a lot with command-line argument processing when an argument takes an value. But I'm fine with redoing this one.